File: dutch national flag problem/dutch.js
//Given an array A[] consisting of only 0s, 1s, and 2s. The task is to write a function that sorts the given array. The functions should put all 0s first, then all 1s and all 2s in last.
function dutch(arr){
    let low=0
    let mid=0
    let high=arr.length-1
    while(mid<=high){
        if(arr[mid]==0){
            let temp=arr[mid]
            arr[mid]=arr[low]
            arr[low]=temp
            low++
            mid++
        }
        else if(arr[mid]==1){
            mid++
        }
        else{
            let temp=arr[mid]
            arr[mid]=arr[high]
            arr[high]=temp
            high--
        }
    }
    return arr
}

File: dutch national flag problem/dutch.py
#Given an array A[] consisting of only 0s, 1s, and 2s. The task is to write a function that sorts the given array. The functions should put all 0s first, then all 1s and all 2s in last.
def dutch(arr):
    low = 0
    mid = 0
    high = len(arr)-1
    while mid <= high:
        if arr[mid] == 0:
            arr[low],arr[mid] = arr[mid],arr[low]
            low += 1
            mid += 1
        elif arr[mid] == 1:
            mid += 1
        else:
            arr[mid],arr[high] = arr[high],arr[mid]
            high -= 1
    return arr
print(dutch([0,1,2,0,1,2]))